quickstart-node/src/services/jwt.service.ts

```diff
@@ -0,0 +1,52 @@
+import { EnvironmentVariablesService, SecretsService } from '@services/monday-code';
+import jwt, { JsonWebTokenError, JwtPayload, TokenExpiredError } from 'jsonwebtoken';
+import { InternalServerError, Unauthorized } from 'http-errors';
+import axios from 'axios';
+import { EnvironmentsKeys, SecretsKeys } from '@shared/monday-auth-consts';
+
+export class JWTService {
+  private constructor() {}
```

Comment:
Do you need a constructor if it's empty?

Reply:
I dont want people to do this: `new JWTService()` that is why I privated the constructor